packages/server/__tests__/addFeatureFlag.test.ts

```diff
@@ -0,0 +1,52 @@
+import {sendPublic, signUp} from './common'
+import ServerAuthToken from '../database/types/ServerAuthToken'
+import encodeAuthToken from '../utils/encodeAuthToken'
+
+const ADD_FEATURE_FLAG = `
+  mutation AddFeatureFlag($emails: [String!], $domain: String, $flag: String!) {
+    addFeatureFlag(emails: $emails, domain: $domain, flag: $flag) {
+      error {
+        title
+        message
+      }
+      users {
+        id
+        featureFlags {
+          poker
+        }
+      }
+    }
+  }
+`
+
+test('Add feature flag by email', async () => {
+  const {email, userId} = await signUp()
+  const authToken = encodeAuthToken(new ServerAuthToken())
+
+  const update = await sendPublic({
+    query: ADD_FEATURE_FLAG,
+    variables: {
+      emails: [email],
+      flag: 'poker'
+    },
+    authToken
+  })
+
+  console.log(JSON.stringify(update, undefined, 2))
```

Comment:
nit: remove?

Reply:
Good catch, I forget once to prefix my debug statements and immediately push these